src/components/Setup/FormatDialog.tsx

```diff
@@ -28,29 +29,34 @@ const FormatDialog: FC<Props> = ({ containsBlockchain, callback }) => {
   return (
     <SetupContainer>
       {step === 1 && (
-        <div className="text-center">
-          <div className="text-center">{t("setup.blockchain_found_short")}</div>
-          <div className="text-center text-sm">
-            {t("setup.blockchain_found_long")}
-          </div>
-          <button
-            onClick={() => handleBlockchain(false)}
-            className="bd-button my-5 p-2"
-          >
-            {t("setup.blockchain_delete")}
-          </button>
-          &nbsp;
-          <button
-            onClick={() => handleBlockchain(true)}
-            className="bd-button my-5 p-2"
-          >
-            {t("setup.blockchain_keep")}
-          </button>
-        </div>
+        <section className="flex h-full flex-col items-center justify-center p-8">
+          <h2 className="m-2 text-center text-lg font-bold">
+            {t("setup.blockchain_found_short")}
+          </h2>
+          <div className="text-sm">{t("setup.blockchain_found_long")}</div>
+          <article className="mt-10 flex flex-col items-center justify-center gap-10 md:flex-row">
+            <button
```

Comment:
nit: There' s no form around these buttons, right? Could those be `a`?

Reply:
Yeah, but it's still a button and it doesn't thange the URL of the page, just changes the component,, so I don't think a `a` would be fitting here